packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -281,12 +282,29 @@ def _init_rate_limiter(self) -> None:
     def _build_messages(
         self, prompt: MultimodalPrompt, system_instruction: Optional[str] = None
     ) -> List[Dict[str, str]]:
+        audio_format = None
         messages = []
-        for parts in prompt.parts:
-            if parts.content_type == PromptPartContentType.TEXT:
-                messages.append({"role": "system", "content": parts.content})
+        for part in prompt.parts:
+            if part.content_type == PromptPartContentType.TEXT:
+                messages.append({"role": "system", "content": part.content})
+            elif part.content_type == PromptPartContentType.AUDIO_FORMAT:
+                audio_format = AudioFormat(part.content)
+            elif part.content_type == PromptPartContentType.AUDIO_BYTES:
+                if not audio_format:
+                    raise ValueError("No audio format provided")
+                messages.append(
+                    {  # type: ignore
+                        "role": "user",
+                        "content": [
+                            {
+                                "type": "input_audio",
+                                "input_audio": {"data": part.content, "format": audio_format.value},
```

Comment:
maybe we should be trying to infer the format from the file headers instead—we'll just do our best effort and fall back to something sensible if the inference doesn't work. We have the bytestring so the headers should be there